test/system/ssh_keys_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws aws_edge baremetal packet
+// +build e2e
+
+package system_test
```

Comment:
This is to satisfy new linter, which IMO we should incorporate also for other code: https://github.com/maratori/testpackage.

Reply:
Have we agreed on using this new linter? I'm not familiar with this tool. Anyway, if that's what this linter is suggesting, I'd say we need to reconsider using it :-/

Reply:
Generally speaking, we should be following best practices and this is one of them. The linter enforces them. If we don't agree to follow some, we make a team decision and we disable some linters. This allows writing more consistent code for everyone involved in the project.

In this particular case, using this package name for unit tests enforces only testing exported functions, which allows to do refactoring in the future. It does not make much sense for e2e tests though, but I don't see a reason why we should make a distinction here.

Reply:
>Generally speaking, we should be following best practices and this is one of them. The linter enforces them.

Right. The relevant Go best practice is here: https://blog.golang.org/package-names
Where is the best practice which says `system_test` is preferred? :-/

>If we don't agree to follow some, we make a team decision and we disable some linters. This allows writing more consistent code for everyone involved in the project.

Following that logic, shouldn't we have a team decision to *add* a linter, too? In any case, IMO the official Go recommendations are much stronger than the opinion of a community member who wrote a linter. Do you think otherwise?

Reply:
I'd go without `_test`.

I see the point of `_test` convention to force only exported functions to be tested (it's briefly mentioned here https://tip.golang.org/cmd/go/#hdr-Test_packages and it's used in the go standard library).

In this case it doesn't really matter because we're in a separate package anyway and we're not doing unit tests.

Reply:
Thanks @iaguis. I wasn't aware `_test` in package names was a thing, and couldn't figure this out from the discussion above. So IIUC `_test` is a **special** case, i.e. there is no recommendation to call a package `cluster_configuration`. Right?

Reply:
Yes, IIUC this is only for tests, not for package names in general.

Reply:
If I should go without `_test` (though I think it does not matter here and it's a good habit to keep it), what should I do with the linter? Do we want to disable the rule globally? Or should each new test file get `//nolint:testpackage`?

Reply:
Just added `//nolint:testpackage` for now.